java_console/io/src/test/java/com/rusefi/native_/JniUnitTest.java

```diff
@@ -27,7 +27,6 @@ public void run() {
 
         assertEquals(TS_FILE_VERSION, (int) getValue(engineLogic.getOutputs(), Sensor.FIRMWARE_VERSION));
 
-        assertEquals(14.0, getValue(engineLogic.getOutputs(), Sensor.TARGET_AFR));
```

Comment:
@mck1117 I am a bit confused. Was this the line which was supposed to cover us from #4826